react/features/calendar-sync/components/ConferenceNotification.native.js

```diff
@@ -0,0 +1,242 @@
+// @flow
+import React, { Component } from 'react';
+import {
+    Dimensions,
+    Text,
+    TouchableOpacity,
+    View
+} from 'react-native';
+import { connect } from 'react-redux';
+
+import { appNavigate } from '../../app';
+import { getURLWithoutParams } from '../../base/connection';
+import { getLocalizedDateFormatter, translate } from '../../base/i18n';
+import { Icon } from '../../base/font-icons';
+
+import styles from './styles';
+
+const ALERT_MILLISECONDS = 5 * 60 * 1000;
+
+type Props = {
+
+    /**
+     * The Redux dispatch function.
+     */
+    dispatch: Function,
+
+    /**
+     * The URL of the current conference without params.
+     */
+    _currentConferenceURL: string,
+
+    /**
+     * The calendar event list.
+     */
+    _eventList: Array<Object>,
+
+    /**
+     * The height of the screen. Needed to be in props to trigger a re-render.
+     */
+    _screenHeight: number,
```

Comment:
What was the alternative to putting this in props? How does the app know to trigger a redux state change, which would trigger mapStateToProps with a new screenHeight, which would cause a re-render?

Reply:
The reason for doing it was to make sure that we update the screen height any time when an ASPECT_RATIO_CHANGE event happens. But since this are is changed in master since then, I just refactored this area